docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -2,8 +2,8 @@
 
 <p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
 
-Are you working with MUI's component libraries? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
+This is a curated list of some of the best apps we've seen that show off what's possible with Material UI. Are you also using it? [Show us what you're building](https://github.com/mui/material-ui/issues/22426)! We'd love to see it.
 
-This is a curated list of some of the best apps we've seen that show off what's possible with MUI's products.
+---
 
-{{"demo": "Showcase.js", "hideToolbar": true, "bg": "inline"}}
+{{"component": "docs/src/modules/components/MaterialShowcase.js"}}
```

Comment:
This new file location feels wrong. We should be able to render the component at the same location as the markdown. Issue created https://github.com/mui/material-ui/issues/42225